_schemaV2.graphql

```diff
@@ -6129,6 +6129,7 @@ type Me implements Node {
   email: String
   followsAndSaves: FollowsAndSaves
   hasCreditCards: Boolean
+  hasPassword: Boolean!
```

Comment:
@zephraph recommended that we check again the naming here. Please feel free to suggest a different one

Reply:
how about ? hasRegisteredUsingEmail

Reply:
This flag does match gravity and its intended purpose is to help make UI decisions based on if the user has a password or not, so it's sort of aptly named. 

I just recommended thinking about the naming in case we don't want to telegraph that particular information. Given as it's only available through `me` though, that probably doesn't matter.